pdb2sql/StructureSimilarity.py

```diff
@@ -322,25 +314,17 @@ def compute_izone(self, cutoff=10.0, save_file=True, filename=None):
         """
 
         sql_ref = interface(self.ref)
-        chains = list(sql_ref.get_chains())
-        if len(chains) != 2:
-            raise ValueError(
-                'exactly two chains are needed for irmsd calculation but we found %d' % len(chains), chains)
 
-        contact_ref = sql_ref.get_contact_atoms(
-            cutoff=cutoff, extend_to_residue=True, chain1=chains[0], chain2=chains[1])
+        contactID_ref = [] # [('X', 2, 'ARG'), ('X', 3, 'GLY')]
+        for chn_grp in chain_pairs:
+            #chn_grp ="AB:C"
+            chns_1, chns_2 = chn_grp.split(':') #chns_1='AB', chns_2='C'
+            for chain1 in [*chns_1]:
+                for chain2 in [*chns_2]:
+                    contact = sql_ref.get_contact_residues(cutoff=cutoff,  chain1= chain1, chain2= chain2)
+                    contactID_ref = contactID_ref + sum(list(contact.values()), []) #flatten the list of lists
```

Comment:
I think we should modify `get_contact_atoms`  so that it can accept multiple chains for chain1 and chain2 argument. (the `get` function already does that so it shouldn't be too problematic). Then we could simply use:

```python
contact_ref = sql_ref.get_contact_atoms(
            cutoff=cutoff, extend_to_residue=True, chain1=list(chns_1), chain2=list(chns_2))
``` 

I can quickly fix that